test/factories/github/team_members.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :github_team_member, class: 'Github::TeamMember' do
-    user_id { SecureRandom.hex }
+    user { create(:user, uid: SecureRandom.hex) }
```

Comment:
```suggestion
    user
```

I'm not sure why you need to specify `uid`here. I think just a normal user is fine.